app/src/main/res/values/styles.xml

```diff
@@ -202,10 +202,25 @@
     <item name="fontFamily">sans-serif-medium</item>
   </style>
 
-  <style name="Heading3" parent="TextView.Common1">
+  <style name="HeadingViewStart2" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif-medium</item>
+    <item name="android:textSize">20sp</item>
+    <item name="fontFamily">sans-serif-medium</item>
+    <item name="android:textAlignment">viewStart</item>
+  </style>
+
+  <style name="HeadingCenter3" parent="TextView.Common1">
```

Comment:
Suggest HeadingViewCenter3 to have parity with the view start variant. Ditto for the others.

Reply:
Done.